src/stylesheets/elements/_headings.scss

```diff
@@ -153,30 +153,49 @@ footer h2 {
   }
 }
 
+/*--- Track Timelines ---*/
+
 .track > {
-  h2, h3, h4, h5, h6 {
-    color: $grey-1;
-    position: relative;
-    line-height: 1.667; // 30px
-    @include epsilon;
-    &:before {
-      z-index: 1;
-      left: -24px;
-      width: 11px;
-      content: '';
-      height: 11px;
-      margin-top: 6px;
-      border-radius: 50%;
-      position: absolute;
-      vertical-align: top;
-      bottom: calc(50% - 5px);
-      background-color: $grey-1;
+  li > {
+    h2, h3, h4, h5, h6 {
+      position: relative;
+      color: $grey-1;
+      line-height: 1.667; // 30px
+      @include epsilon;
+      &:before {
+        z-index: 1;
+        left: -24px;
+        width: 11px;
+        content: '';
+        height: 11px;
+        margin-top: 6px;
+        border-radius: 50%;
+        position: absolute;
+        vertical-align: top;
+        top: 4px;
+        background-color: $grey-1;
+      }
+    }
+    h2 + h2, h3 + h3, h4 + h4, h5 + h5 {
+      margin-top: $base-spacing-unit-large;
     }
   }
-  h2 + h2, h3 + h3, h4 + h4, h5 + h5 {
-    margin-top: $base-spacing-unit-large;
+  li:first-child > {
+    h2, h3, h4, h5, h6 {
+      margin-top: 0;
+    }
   }
 }
+
+.track__light {
```

Comment:
Can you rename this to `.track--light`? Then we can add it as an extra class which makes it light. Then it can be similar to the way we are handling lists. For example, when we handle a 3 columned block-list, we would mark it up as `.list--block.list--3`.

Reply:
As discussed, we'll have a collective chat about this - if a new track--light component is needed I'll create later. I have removed track--light from code base for now.